docker-compose.yml

```diff
@@ -34,9 +34,9 @@ services:
         image: silintl/php7:latest
         volumes:
           - .:/data
-        depends_on:
-            - db
-            - broker
+#        depends_on:
+#            - db
+#            - broker
```

Comment:
Why this removal? (I ask because it looks, from line 41, like this does depend on broker.)

Reply:
not sure, uncommented.